ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/routing.j2

```diff
@@ -36,6 +36,8 @@
 {%     include 'documentation/static-routes.j2' %}
 {## IPv6 Static Routes #}
 {%     include 'documentation/ipv6-static-routes.j2' %}
+{## IPv6 Neighbors #}
```

Comment:
add to conditions at the start of this file.

Reply:
fixed